puns.json

```diff
@@ -267,6 +267,33 @@
     "punchline": "A: A*"
   },
   {
+    "pun": "Q: So, where did you two met?",
+    "punchline": "A: Github"
+  },
+  {
+    "pun": "Q: Who is your boyfriend?",
+    "punchline": "A: I'll leave the relations to the database"
+  },
```

Comment:
I didn't think these were particularly funny, could you remove them from the PR please?